spec/features/admin_views_all_groups_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+feature "Admin can see all the groups in the groups/index page" do
+  scenario "successfully" do
+    visit groups_path
```

Comment:
I would create a group in this spec, and check that I can see its details

Reply:
Done in sha: `67fc3ea`